synapse/lib/ast.py

```diff
@@ -1441,23 +1463,42 @@ class LiftFormTagProp(LiftOper):
 
     async def lift(self, runt, path):
 
-        form, tag, prop = await self.kids[0].compute(runt, path)
+        formname, tag, prop = await self.kids[0].compute(runt, path)
 
-        if not runt.model.form(form):
-            raise self.kids[0].kids[0].addExcInfo(s_exc.NoSuchForm.init(form))
+        form = runt.model.form(formname)
+        if not form:
+            forms = runt.model.formsbyiface.get(formname)
+
+            if forms is None and formname.endswith('*'):
+                pref = formname[:-1]
+                forms = []
+                for form in runt.model.forms:
+                    if form.startswith(pref):
+                        forms.append(form)
```

Comment:
Thinking out loud - I'm curious if we'd want to sort this list of forms or not.

Reply:
Yeah that would probably be a good idea for consistency.